addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -350,6 +351,18 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
                 .attr('data-editor-message-default', true)
                 .attr('data-editor-message', this.env._t('DRAG BUILDING BLOCKS HERE'));
         $wrap.filter(':empty').attr('contenteditable', false);
+        for (let htmlEl of $wrap.not("[placeholder]").filter('[data-oe-sanitize="no_block"]')) {
```

Comment:
We can change that later, it's not the core purpose of this task, but this could also have been applied to empty `text` field which don't have a placeholder, right?
Because right now, if we take your example of partner description on a partner page, and go to the root page of partners /partners, the same field will not show a placeholder.

Basically what @pierre-pde illustrated [here](https://github.com/odoo/odoo/pull/120588#issuecomment-1580151525).

It's worth considering in a next task.

cc @pierre-pde